docs/guides/open-api.md

```diff
@@ -0,0 +1,18 @@
+# OpenAPI
+
+Amadeus for Developer provides a set of Self Service APIs which was implemented
```

Comment:
Amadeus for Developer -> Amadeus for Developers 
suggestion: Self Service -> Self-Service (for consistency)